yaks/yaks.gemspec

```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'mutant-rspec' if RUBY_VERSION > '1.9'
   gem.add_development_dependency 'mutant'       if RUBY_VERSION > '1.9'
   gem.add_development_dependency 'rspec-its'
+  gem.add_development_dependency 'rspec-mocks'
```

Comment:
yaks has been using `bogus` instead of `rspec-mocks`. Maybe you could have a look if you could solve things with `bogus` as well? Feel free to bother @chastell if you need help with that :P